runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -973,20 +1002,19 @@ J9::TransformUtil::canFoldStaticFinalField(
       static const bool disableAggressiveVarHandleSFFF =
          feGetEnv("TR_disableAggressiveVarHandleSFFF17") != NULL;
 
-      if (!disableAggressiveVarHandleSFFF && cpIndex >= 0)
+      if (!disableAggressiveVarHandleSFFF)
```

Comment:
Is this change to omit `&& cpIndex >= 0` intentional ? I ask, since it seems to be along different lines to the rest of the commit, that appears more focused on the new disable option.

Reply:
Yes, there's an assertion above that checks to make sure we have a valid `cpIndex` because it's needed to get the name and signature of the field. So now it's guaranteed at this point